### DISCUSSION TITLE
How to implement the basic graphql code gen modules in github CI/CD pipeline

### DISCUSSION THREAD

**Discussion by ankit9787** [dotansimha/graphql-code-generator#9583] [Docs Discussions]
My ask is-
Lets take a basic case of Apollo GraphQL Server project-
So steps mentioned are , If we have schema file and few dependencies installed then run graphql codegen command and resolver-types files will be generated. So how to achieve the same via CI/CD pipeline.
Any document for its integration in GitHub or any CI/CD pipeline ?

**Comment by raajnadar** [comment]
Even I was looking for a document for this with the GitHub Actions
